catalog/Dockerfile

```diff
@@ -7,8 +7,9 @@ ENV LANG=C.UTF-8
 RUN apt-get update && apt-get install -y \
     nginx \
     curl \
-    gettext
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+    gettext \
+    libssl-dev
+RUN curl -sL https://deb.nodesource.com/setup_9.x | bash -
```

Comment:
Why are you bumping node version?

Reply:
docker-compose fails on windows with 8 node version with this `package.json`

Reply:
Isn't this a Dockerfile? Isn't it strange that the node version inside the dockerfile would cause OS-specific errors? Can you post the error you're seeing?

Reply:
Docker for windows uses `hypre-V` by default.
Actually, 8th version of node cause freeze `npm install` on windows. I assumed that it caused by sim-links issue, because `hyper-V + vagrant + npm` failed for me previously and `npm install --no-bin-links` did the trick. So I tried to change .Dockerfile to install without sim-links, but it effected `npm build` tasks. Since there are quite complicated tasks structure inside `package.json`. After little google this issue, I tried update node version and it fix docker build process. If node version upgrade is not suitable solution, I can try upgrade `npm` only, this might resolve an issue too.

Reply:
Could you try just updating npm? I don't want to have two Node versions if we don't need to.